src/main/kotlin/com/kotlin/boardproject/domain/comment/CommentController.kt

```diff
@@ -1,21 +1,21 @@
-package com.kotlin.boardproject.controller
+package com.kotlin.boardproject.domain.comment
 
-import com.kotlin.boardproject.auth.LoginUser
-import com.kotlin.boardproject.common.util.log
-import com.kotlin.boardproject.dto.comment.BlackCommentRequestDto
-import com.kotlin.boardproject.dto.comment.BlackCommentResponseDto
-import com.kotlin.boardproject.dto.comment.CancelLikeCommentResponseDto
-import com.kotlin.boardproject.dto.comment.CreateCommentRequestDto
-import com.kotlin.boardproject.dto.comment.CreateCommentResponseDto
-import com.kotlin.boardproject.dto.comment.DeleteCommentRequestDto
-import com.kotlin.boardproject.dto.comment.DeleteCommentResponseDto
-import com.kotlin.boardproject.dto.comment.LikeCommentResponseDto
-import com.kotlin.boardproject.dto.comment.UpdateCommentRequestDto
-import com.kotlin.boardproject.dto.comment.UpdateCommentResponseDto
-import com.kotlin.boardproject.dto.common.ApiResponse
-import com.kotlin.boardproject.service.CommentService
-import com.kotlin.boardproject.service.NotificationService
-import com.kotlin.boardproject.service.SseService
+import com.kotlin.boardproject.global.annotation.LoginUser
```

Comment:
🚫 **[ktlint]** <standard:import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between with "java", "javax", "kotlin" and aliases in the end